GE_Requirements.mediawiki

```diff
@@ -2,24 +2,43 @@
 * Pull Requests MUST include not only code changes but also test changes and documentation changes.
-* Pull Requests MUST be public and open during a reasonable period of time so that the community can check it and contribute if necessary
+* Pull Requests MUST be public and open during a reasonable period of time so that the community can check them and contribute if necessary
 * Pull Requests need to be approved (using the standard Github function or a 'LGTM' comment) by all the reviewers involved, i.e. all people that gave their feedback on it.
-* Pull Request approval MUST be issued by a thrid party. Nobody can auto-merge their contributions.
+* Pull Request approval MUST be issued by a third party. Nobody can auto-merge their contributions.
 * Before merging an open Pull Request Travis status MUST be green. 
 
 
 == Publication Requirements ==
 
 * The SCM Tool MUST be [https://github.com Github.com]. 
-* Github MUST be used during the <b>whole development lifecycle</b>. See [[development.mediawiki#Development_Lifecycle|Development Lifecycle]] which describes a set of best practices for collaborative open source development.
-* See [[GE_Checklist.mediawiki|GE Publication Checklist]] for a list of tasks to be done in order to publish a new GE.
+* Github MUST be used during the <b>whole development lifecycle</b>. See [[development.mediawiki#Development_Lifecycle|Development Lifecycle Recommendations in FIWARE]]
+which describes a set of best practices for collaborative open source development.
+* See [[GE_Checklist.mediawiki|Checklist for publishing FIWARE Generic Enablers]] for a list of tasks to be done in order to publish a new GE.
+
+== Requirements regarding reference to FIWARE ==
+
+The README.md file of the GitHub repo, or several GitHub repos, associated to a FIWARE GE, '''MUST''' include an Introduction/About section at the beginning which includes:
+
+* the following paragraph clearly showing that the product has been labeled as a FIWARE GE and introducing FIWARE: 
+::'<name of the product> is a FIWARE Generic Enabler. Therefore, it  can be integrated as part of any platform “Powered by FIWARE”. FIWARE is a curated framework of open source platform components which can be assembled together with other third-party platform components to accelerate the development of Smart Solutions. You can find more info at the [https://deveopers.fiware.org FIWARE developers] website and the [https://fiware.org FIWARE] website.
+The complete list of FIWARE GEs and Incubated FIWARE GEs can be found at the [https://catalogue.fiware.org FIWARE Catalogue]'
+::
+* the [[project_badges.mediawiki|Project Badges]] which are stated as mandatory
+* what major.minor release of FIWARE each release of the product is compatible with.
+* The release note for each versioned release should include a list of the latest version of interoperable enablers it has been tested against as follows: 
```

Comment:
To avoid doubt the **Releases** section of the same page should probably have an additional bullet, something like:

> * Every release **MUST** have an associated release note

And then the two bullet points about the **contents** of the release note are moved under it.

As @fgalan says, switching requirements between readme and release note is confusing, but a simple change of ordering should fix this.